src/widgets/conditions/conditions-detailed-summary.component.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { match } from "react-router";
+import { performPatientConditionSearch } from "./conditions.resource";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import dayjs from "dayjs";
+import styles from "./conditions-detailed-summary.css";
+import SummaryCard from "../cards/summary-card.component";
+import { useCurrentPatient } from "@openmrs/esm-api";
+
+export default function ConditionsDetailedSummary(
+  props: conditionsDetailedSummaryProps
+) {
+  const [patientCondition, setPatientCondition] = React.useState(null);
+  const [isLoadingPatient, patient, patientUuid] = useCurrentPatient();
+
+  React.useEffect(() => {
+    if (!isLoadingPatient && patient) {
+      const abortController = new AbortController();
+
+      performPatientConditionSearch(
+        patient.identifier[0].value,
+        abortController
+      )
+        .then(condition => setPatientCondition(condition))
+        .catch(createErrorHandler());
+
+      return () => abortController.abort();
+    }
+  }, [isLoadingPatient, patient]);
+
+  function displayConditions() {
+    return (
+      <SummaryCard
+        name="Conditions"
+        match={props.match}
+        styles={{ width: "100%" }}
+        addBtnUrl={`/patient/${patientUuid}/chart/condition/add`}
+      >
+        <table className={styles.conditionTable}>
+          <thead>
+            <tr>
+              <td>CONDITION</td>
+              <td>ONSET DATE</td>
+              <td>STATUS</td>
+              <td></td>
+            </tr>
+          </thead>
+          <tbody>
+            {patientCondition &&
+              patientCondition.entry.map(condition => {
+                return (
+                  <React.Fragment key={condition.resource.id}>
+                    <tr>
+                      <td className={"omrs-bold"}>
+                        {condition.resource.code.text}
+                      </td>
+                      <td>
+                        <div className={`${styles.alignRight}`}>
+                          {dayjs(condition.resource.onsetDateTime).format(
+                            "MMM-YYYY"
+                          )}
+                        </div>
+                      </td>
+                      <td>
+                        <div
+                          className={`${styles.centerItems} ${styles.alignLeft}`}
+                        >
+                          <span>
+                            {capitalize(condition.resource.clinicalStatus)}
+                          </span>
+                        </div>
+                      </td>
+                      <td>
+                        <svg className="omrs-icon" fill="rgba(0, 0, 0, 0.54)">
```

Comment:
```suggestion
                        <svg className="omrs-icon" fill="var(--omrs-color-ink-low-contrast)">
```
recommend we use the style guide colors [style guide](https://styleguide.openmrs.org/?path=/story/openmrs-styleguide--colors)

Reply:
Done